trento/adoc/trento-mcp-install-systemd.adoc

```diff
@@ -0,0 +1,156 @@
+include::product-attributes.adoc[]
+
+[[sec-trento-mcp-server-systemd]]
+==== systemd {tr_mcp} deployment
+
+A systemd-based installation of the {tr_mcp} using RPM packages can be performed manually on the latest supported versions of {sles4sap}, from 15 SP4 up to 16.0. For installations on service packs other than the current one, make sure to update the repository URL as described in the relevant notes throughout this guide.
+
+include::partials/supported-versions.adoc[]
+
+===== Installing {tr_mcp} using RPM packages
```

Comment:
We have split this procedure is several little subsections: Fist the zypper install step, then the configuration steps, then the service start steps and finally the service verification steps. I think it would give a the user a better sense of beginning and end if all the stpes where in the same block. No splits.